frontend/public/components/build.tsx

```diff
@@ -480,6 +489,18 @@ export const BuildsList: React.SFC = (props) => {
         transforms: [sortable],
         props: { className: tableColumnClasses[3] },
```

Comment:
Let us remove the Created column if we show the Start time. We don't show Created anywhere else.

Reply:
Removed the Created column.